modules/mauth-test-utils/src/main/resources/keys/fake_publickey2.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAs3zgudWufUW/UFoQtxfB
+ePRDqc63RdpvuBILKHNFAYpxxuSRMzQUKSrsvXpWf983/kTtvXBrj4Hm6qWC9fqG
+Kh/qDTnjamXUzxg6utFbcrOrc4hsfvfRjEaymclTPLXRa9+iD3vTzDBqS85eDFDm
+5hTpPwc3C/7+iw0cfWqwP7jJsn5Ovul50UOuR+czeBWuQXZPh21nUpEI7ZW8U3W2
+lmO2DrSez2FVtW3Oiek/AQz+uOtlNSPoJcH06O2dlh7AAF8c3eFTaYH4gBTFzcBU
+p3BneZ2cr0e5m5eVRcCOFqDGQF3YJQT2I1EpMecRWo7UW48mF0wzVj/mK1rQUAeb
+gQIDAQAB
+-----END PUBLIC KEY-----
```

Comment:
Any reason why we need another public key?

Reply:
We try to sync up the test fixture with other platform (Ruby version is standard).  I use the same test data as Ruby and Python in https://github.com/mdsol/mauth-jvm-clients/blob/384557bc88e9958b3295049bff5b310494ccd739/modules/mauth-authenticator/src/test/scala/com/mdsol/mauth/RequestAuthenticatorBaseSpec.scala#L140-L151
The new public key (fake_publickey2.pem) is for the private key which is used to generate the signature  CLIENT_REQUEST_SIGNATURE_BINARY_V1 https://github.com/mdsol/mauth-jvm-clients/blob/384557bc88e9958b3295049bff5b310494ccd739/modules/mauth-authenticator/src/test/scala/com/mdsol/mauth/RequestAuthenticatorBaseSpec.scala#L144-L150

Reply:
Fair enough